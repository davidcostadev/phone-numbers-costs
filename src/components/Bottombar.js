import React from 'react';
import PropTypes from 'prop-types';

const Bottombar = ({ children }) => (
  <div className="bottombar">
    {children}
  </div>
);

Bottombar.propTypes = {
  children: PropTypes.any.isRequired,
};

export default Bottombar;
