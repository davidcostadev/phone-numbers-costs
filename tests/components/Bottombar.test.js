import React from 'react';
import renderer from 'react-test-renderer';
import Bottombar from '../../src/components/Bottombar';

describe('Bottombar Component', () => {
  describe('Render', () => {
    it('default', () => {
      const tree = renderer.create((
        <Bottombar>
          <div>children</div>
        </Bottombar>
      )).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
