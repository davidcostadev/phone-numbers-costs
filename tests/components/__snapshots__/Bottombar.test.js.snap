// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bottombar Component Render default 1`] = `
<div
  className="bottombar"
>
  <div>
    children
  </div>
</div>
`;
